src/structures/interfaces/InteractionResponses.js

```diff
@@ -113,6 +116,7 @@ class InteractionResponses {
    *   .catch(console.error);
    */
   editReply(options) {
+    if (!this.deferred && !this.replied) throw new Error('INTERACTION_NO_RESPONSE');
```

Comment:
```suggestion
    if (!this.deferred && !this.replied) throw new Error('INTERACTION_NOT_REPLIED');
```
shouldn't this be this instead?

Reply:
Yeah thats what I get for renaming stuff, thanks!